demos/projects/PC/linux/tests/aux_needed_functions.c

```diff
@@ -0,0 +1,256 @@
+/* Standard includes. */
```

Comment:
So my read of this is that this is a bunch of FreeRTOS land stuff we need to mock out to get a UT building correctly, right?

If so could we replace "aux" with "mock" or some such?  "aux" makes it sound like they're legit helpers that we need to make work, whereas this is just a compile thing.

Also recommend that we put comments at top of file indicating the intent of this file.

Reply:
I also like the `/* Needed for compilation */` comment above one of the functions and would consider for all of them.

If I'm in a source editor and trying to lookup implementation to say `SystemWhateverFoo`, I'll jump to middle of file | the function itself and having the brief preamble above it is helpful so I know to look at next one.

Reply:
done